app/include/zmk/endpoints.h

```diff
@@ -13,3 +13,4 @@ int zmk_endpoints_toggle();
 enum zmk_endpoint zmk_endpoints_selected();
 
 int zmk_endpoints_send_report(uint16_t usage_page);
+int zmk_endpoints_send_mouse_report();
```

Comment:
Why not just use the existing API and pass the button page id?

Reply:
Unlike keyboards, mice don't have their own usage page in the HID spec. Instead mice is usage 0x02 inside the Generic Desktop (GD) usage page.

Using the generic usage page to signify sending a mouse report seems unintuitive to me, and using the actual usage (0x02) would collide with the "simulation" usage page.

I do agree that adding a separate API surface for mice is also not ideal, but unifying the two would, to my knowledge, require changes to the existing API.

Reply:
We could do that. I think it's maybe a little odd since we're not telling the api which device we want to send a report for (in case you also had a button subusage inside the keyboard), but there's no technical problem.

Reply:
Hmmm... Let me ponder that a bit more.

Reply:
I'm having a hard time thinking of why we would have that subusage any time soon. If you can give me a compelling use case, I'm open to changing toward your proposed approach, otherwise I'd say we stick with the existing API/approach.